notebooks/notebooks/iclr2022/win-rate-vs-visits.ipynb

```diff
@@ -0,0 +1,460 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        df = data for adversary visit sweep experiment
</pre><p><code>df</code> and <code>df2</code> are not very informative names!  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>